### ISSUE TITLE
The polyfill functions in `@paprika/helpers` should be in a separated package as a peer dependency 

### ISSUE THREAD

**Issue by allison-c** [acl-services/paprika#937]
# Bug Report

E.g `elementScroll.js` is the polyfill for `Element.scroll(), Element.scrollTo(), Element.scrollBy()`, but it's very common that a consuming app has paprika components in different ages, most of them will have different versions of `@paprika/helpers` package, which means we'll have multiple versions of `elementScroll.js` . 

However, each polyfill is an IIFE, the later executed `elementScroll.js` will override the previous one. `Element.scroll(), Element.scrollTo(), Element.scrollBy()` will keep changing until everything loaded and executed. It will be very hard to debug when needed.

## Expected behavior

Polyfills should be peer dependencies, or maybe because IE 11 will be unsupported soon, we can remove them?

## Current behavior

Multiple versions of polyfills in consuming apps.

## Screenshots / Gifs / Codepens
*Include media to help illustrate the bug. Gifs help a lot!*

## Additional context

cc @AndreyChernykh

**Comment by nahumzs** [comment]
I think would be better to have specific packages for polyfills? also we are no longer supporting ie11 🤔 would this still be relevant?

**Comment by allison-c** [comment]
> I think would be better to have specific packages for polyfills? also we are no longer supporting ie11 🤔 would this still be relevant?

Yeah we definitely don't need this after September, maybe we need some cleanup at that time

**Comment by AndreyChernykh** [comment]
@nahumzs that depends on a polyfill. We'll need to check the compatibility for each feature and decide if we still need some polyfills or not.

**Comment by oscarkwan** [comment]
@allison-c Do you think this is still needed?

**Comment by oscarkwan** [comment]
Hi @nahumzs  LOL

**Comment by allison-c** [comment]
haha hi Nahum!

**Comment by oscarkwan** [comment]
Sounds like this is still relevant for cleaning up the polyfills! Will leave this :)
